cog.c

```diff
@@ -398,6 +398,11 @@ main (int argc, char *argv[])
         g_set_application_name ("Cog");
     }
 
+#if COG_USE_WEBKITGTK
+    // Workaround for https://bugs.webkit.org/show_bug.cgi?id=150303
```

Comment:
It *is* a workaround because calling `gtk_init()` is not expected to be needed when using [GtkApplication](https://developer.gnome.org/gtk3/stable/GtkApplication.html#gtk-application-new), as stated in the documentation:

> When using GtkApplication, it is not necessary to call gtk_init() manually. It is called as soon as the application gets registered as the primary instance.
>
> Concretely, gtk_init() is called in the default handler for the “startup” signal. Therefore, GtkApplication subclasses should chain up in their “startup” handler before using any GTK+ API.

For the record, our `CogLauncher` subclass is already doing the required chaining-up in `cog_launcher_startup()` as instructed.

Reply:
Another note: If you look at the [linked bug in the WebKit Bugzilla](https://bugs.webkit.org/show_bug.cgi?id=150303), you will notice that this is due to a regression: it used to be the case that fetching the default context before `gtk_init()` used to work fine, but not anymore. I don't know so far whether the regression is in WebKitGTK+ or in GTK+ itself — but *there is* a regression.